.github/workflows/docs.yml

```diff
@@ -1,29 +1,35 @@
-name: Deploy Docs
+name: Deploy Docs to GH Pages
 
 on:
   push:
     branches:
       - main
 
 jobs:
-  deploy_docs:
+  deploy:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
-      - name: Setup node 😻
-        uses: actions/setup-node@v3
-        with:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
           node-version-file: '.node-version'
           cache: 'yarn'
 
-      - name: Install Dependencies ⬆️
-        run: yarn install --immutable
+    - name: Install Dependencies
+      run: yarn install --immutable
+
+    - name: Build EDS
+      run: yarn build
 
-      - name: Build Dist 🔧
-        run: yarn run build
+    - name: Build Storybook
+      run: yarn build:storybook
 
-      - name: Deploy Storybook Docs 🚀
-        run: yarn run deploy:docs
-        env:
-          GH_TOKEN: '${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}'
+    - name: Deploy Storybook
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        publish_dir: ./storybook-static
```

Comment:
the naming is a bit awkward here... this should be read as `dir_to_publish` or similar; this *isn't* where the docs will be published to, and won't end up in the path